cmd/kubeadm/app/cmd/join.go

```diff
@@ -282,15 +283,20 @@ func NewJoin(cfgPath string, args []string, defaultcfg *kubeadmapiv1beta1.JoinCo
 		return nil, err
 	}
 
+	initcfg, err := fetchInitConfigurationFromJoinConfiguration(internalcfg)
```

Comment:
@ereslibre please fix the fact we are doing TLSbootstrap twice

Reply:
Yes, I will have a look at this today. Thanks!